packages/flutter/lib/src/material/divider.dart

```diff
@@ -138,37 +161,57 @@ class Divider extends StatelessWidget {
 ///  * [ListView.separated], which can be used to generate vertical dividers.
 ///  * <https://material.io/design/components/dividers.html>
 class VerticalDivider extends StatelessWidget {
-  /// Creates a material design divider.
+  /// Creates a material design vertical divider.
   ///
-  /// The width must be positive.
+  /// The [width], [thickness], [indent], and [endIndent] must be null or
```

Comment:
The terminology for values that can be null or >= 0 is a little tricky because zero. In most cases we have been writing "null or non-negative". I don't think that's more correct or even shorter, but consistency.

Reply:
done